.circleci/config.yml

```diff
@@ -1,54 +1,58 @@
 version: 2
 
 jobs:
-  lint:
+  buildImage:
     docker:
-      - image: circleci/node:10.15
+      - image: web3f/ci-commons:v1.7.1
     steps:
       - checkout
-      - run: yarn
-      - run: yarn lint
+      - setup_remote_docker
+      - run:
+          command: /scripts/build-image.sh web3f/cla-assistant .
 
-  buildImage:
+  publishImage:
     docker:
       - image: web3f/ci-commons:v1.7.1
     steps:
       - checkout
       - setup_remote_docker
       - run:
-          command: |
-            /scripts/build-image.sh web3f/cla-assistant .
+          command: /scripts/publish-image.sh web3f/cla-assistant
 
-  integrationTests:
+  testChart:
     docker:
       - image: web3f/ci-commons:v1.7.1
     steps:
       - checkout
-      - setup_remote_docker
       - run:
-          description: run integration tests
           command: |
-            /scripts/integration-tests.sh
+            helm lint ./charts/cla-assistant/.
```

Comment:
I think we should leave the lint command here and maybe an additional template command you mentioned before, but for install and test we need the integration test setup, remote docker and a running cluster. So IMO we can remove them from here and create a `./scripts/integration-tests.sh` file in this repo, it will be used automatically in the `integrationTests` step, wdyt?